addons/website/static/src/scss/website.scss

```diff
@@ -72,6 +73,37 @@ $-seen-urls: ();
         $i: $i + 1;
     }
 
+    @include print-variable('paragraph-margin-top', $paragraph-margin-top);
+    @include print-variable('h2-line-height', $h2-line-height);
+    @include print-variable('h3-line-height', $h3-line-height);
+    @include print-variable('h4-line-height', $h4-line-height);
+    @include print-variable('h5-line-height', $h5-line-height);
+    @include print-variable('h6-line-height', $h6-line-height);
+    @include print-variable('h1-font-size', $h1-font-size);
+    @include print-variable('h2-font-size', $h2-font-size);
+    @include print-variable('h3-font-size', $h3-font-size);
+    @include print-variable('h4-font-size', $h4-font-size);
+    @include print-variable('h5-font-size', $h5-font-size);
+    @include print-variable('h6-font-size', $h6-font-size);
+    @include print-variable('h2-font', o-website-value('h2-font') or o-website-value('headings-font') or o-website-value('font'));
+    @include print-variable('h3-font', o-website-value('h3-font') or o-website-value('headings-font') or o-website-value('font'));
+    @include print-variable('h4-font', o-website-value('h4-font') or o-website-value('headings-font') or o-website-value('font'));
+    @include print-variable('h5-font', o-website-value('h5-font') or o-website-value('headings-font') or o-website-value('font'));
+    @include print-variable('h6-font', o-website-value('h6-font') or o-website-value('headings-font') or o-website-value('font'));
```

Comment:
Why not using the `$o-theme-hx-font` variables which are already handle the fallback?

Reply:
Because they contain the list of fallback fonts and they therefore do not match the possible values inside the font selection dropdowns. I used this `or` sequence in order to have the single font name in the value.

Reply:
@qsm-odoo Change pushed. <https://github.com/odoo/odoo/compare/7b265fdc75c1f3fd52535bb8005b68c4dfcdafc1..fac6779840658b3478641ecfdd7dbdc7b5315984>